package.json

```diff
@@ -28,6 +29,23 @@
     "lint": "tslint -c tslint.full.json --project tsconfig.json --type-check --exclude \"examples/**/*\" --exclude \"**/node_modules/**\" --exclude \"**/*.d.ts\" \"**/*.ts\"",
     "lint:fix": "npm run lint -- --fix",
     "test": "mocha --opts test/mocha.opts.root \"packages/*/test/**/*.ts\"",
-    "posttest": "npm run lint"
+    "posttest": "npm run lint",
+    "coverage": "nyc npm test"
+  },
+  "nyc": {
+    "include": [
+      "packages/*/src/**/*.ts"
+    ],
+    "extension": [
+      ".ts"
+    ],
+    "require": [
+      "ts-node/register"
+    ],
+    "reporter": [
+      "html",
+      "lcov"
+    ],
+    "report-dir": "./test_results/coverage"
```

Comment:
FWIW, we usually use the default `./coverage` directory for the output.

The thing I dislike about `test_results` most is that bash auto-completion of directories does not work well for that name, because we have both `test` and `test_results` candidates to pick from.